pkg/kubectl/cmd/annotate.go

```diff
@@ -92,6 +93,7 @@ var (
 )
 
 func NewCmdAnnotate(f cmdutil.Factory, out io.Writer) *cobra.Command {
+	LoadTranslations()
```

Comment:
If you do it this way we will have to load all translation files from disk for every single command ever run, even if it uses no actual translation output.